src/lib/github.ts

```diff
@@ -66,12 +66,27 @@ export const getRelease = (
   tag: string
 ): Promise<string> => {
   const client = gh.client(token, clientOptions);
-  return client
-    .release(name, `tags/${tag}`)
-    .infoAsync()
-    .then((release: Array<{[key: string]: string}>) => {
-      return release[0].tag_name;
-    });
+  return new Promise((resolve, reject) => {
+    client.get(
+      `/repos/${name}/tags`,
+      {per_page: 100},
+      (err: Error, code: number, resp: [{name: string}]) => {
+        if (err) {
+          return reject(err);
+        } else if (code !== 200)
+          return reject(Error(`unexpected http code = ${code}`));
```

Comment:
`Error()` is a shorthand for `new Error()`.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Error

so, it's just stylistic.

---

> Would rather prepend the message and re-reject the existing exception

If an `err` is populated by `client.get` it gets rejected on line `75`.  In the case of non-200 status codes, there's no error object, just a status code other than 200 ... I think I'm not following.